### PR TITLE
Expand JIRA issues as they are mentioned

### PR DESCRIPTION
**Pull request by Kegsay**
Based off https://github.com/matrix-org/go-neb/pull/21

### CODE REVIEW

File: src/github.com/matrix-org/go-neb/services/jira/jira.go [hunk 1]
```diff
@@ -174,6 +261,20 @@ func (s *jiraService) projectToRealm(userID, pkey string) (*realms.JIRARealm, er
...
+	//   "Flibble Wibble [P1, In Progress]"
+	status := html.EscapeString(issue.Fields.Status.Name)
+	if issue.Fields.Resolution != nil {
+		status = fmt.Sprintf("%s (%s)",
+			status, html.EscapeString(issue.Fields.Resolution.Name))
+	}
+	return fmt.Sprintf("%s [%s, %s]",
+		html.EscapeString(issue.Fields.Summary),
+		html.EscapeString(issue.Fields.Priority.Name),
+		status)
```
Comment (NegativeMjark) [inline src/github.com/matrix-org/go-neb/services/jira/jira.go]: I'm not sure about the dangling ')'

Comment (Kegsay) [inline src/github.com/matrix-org/go-neb/services/jira/jira.go]: Bah thought I got them all, thanks!

**Comment by NegativeMjark** [comment]
Apart from the dangling ')' LGTM.

At somepoint we might want to think about the utility of the error messages. I suspect that the error messages will need to include enough information for the end user to correct their mistakes and typos and I don't think that the current messages give enough information for them to do so.

But I'm not sure how we want to go about doing that, So I'm happy to leave them as they are for now.

**Comment by Kegsay** [comment]
Agreed, there's been places where I've wanted to return more human friendly errors but not had the right error bubbling to do it.

**Pull request merged**